40K/ammo.json

```diff
@@ -65,7 +65,7 @@
     "id": "promethium",
     "type": "AMMO",
     "name": "promethium",
-    "description": "promethium",
+    "description": "A highly volatile fuel used in Imperial incendiary weapons, that ignites on exposure to oxygen.",
```

Comment:
A hypergolic fuel ignites when mixed with O2. Promethium has to be ignited but that's a small gripe.